server/src/routes/api/v1/recipes/recipe.route.ts

```diff
@@ -17,14 +17,14 @@ export class RecipeRoute {
     this._router.post("/",
       this._authValidationHandler.checkToken,
       this._authValidationHandler.checkUser,
-      this._authValidationHandler.nonRestricted,
+      //this._authValidationHandler.nonRestricted,
       this._modelValidationHandler.validate,
       this._recipeHandler.save);
 
     this._router.get("/:id",
       this._authValidationHandler.checkToken,
-      this._authValidationHandler.checkUser,
```

Comment:
What do you think about this? This seems to be a valid approach since it is suggested in a lot of stackoverflow answers